etc/kayobe/kolla/globals.yml

```diff
@@ -31,6 +31,11 @@ prometheus_node_exporter_extra_volumes:
   - "textfile:/var/lib/node_exporter/textfile_collector"
 prometheus_node_exporter_cmdline_extras: "--collector.textfile.directory=/var/lib/node_exporter/textfile_collector"
 
+# Set Ceph manager exporter endpoints
+prometheus_ceph_mgr_exporter_endpoints:
+{% for host in groups['mgrs'] %}
+  - "{{ host }}:9283"
```

Comment:
This uses the Ansible `inventory_hostname`, which is not guaranteed to be resolvable. You could do this:

```
 - {{ admin_oc_net_name | net_ip(host) }}:9283
```